src/data/textures.inc.c

```diff
@@ -1,671 +1,1212 @@
+/**
+ * @file textures.inc.c
+ * @brief Texture data.
+ * start in ram 0x801978D0
+ * look like link to code_80091750
+ */
 #include "textures.h"
 
-// 0x801978D0
+/**
+ * @brief MkTexture of a tiny font texture of number 0
+ *
+ */
 MkTexture D_02000000[2] = {
```

Comment:
Wonder if perhaps these don't need comments and just names.

But merge cause it's fine like this too.

Reply:
when the extrat_all are not enable he all with comment or this include .h file but he are needed to have struct so we need to comment all variable and function to disable extract_all and have less redundencies